### PR TITLE
Store database credentials in k8s secret

### PR DESCRIPTION
**Pull request by anr-tonic**
Currently, this PR doesn't mess w/ stdin echoing because we're unable to figure out how to properly handle it in a cross platform way that also doesn't involve a nightmare.

### CODE REVIEW

File: cli/cmd/timstall/main.go [hunk 1]
```diff
@@ -479,3 +487,50 @@ func (t timDiagnosticsWriter) WriteTo(w io.Writer) (int64, error) {
...
+		return nil, err
+	}
+
+	secretName := ti.TimothyDbSecretName
+	secret, err := client.CoreV1().Secrets(namespace).Get(ctx, secretName, metav1.GetOptions{})
+
+	if err == nil {
+		dbAccess := &dbAccessValues{}
+
+		if username, ok := secret.Data["TIM_DB_USERNAME"]; ok {
```
Comment (anr-tonic) [inline cli/cmd/timstall/main.go:519]: k8s client handles base64 encode/decode automatically. Any b64 references in this PR are vestigial from me thinking I had to handle it myself.

Comment (anr-tonic) [inline cli/cmd/timstall/main.go:519]: Should promote these to consts somewhere.

File: cli/cmd/timstall/main.go [hunk 2]
```diff
@@ -479,3 +487,50 @@ func (t timDiagnosticsWriter) WriteTo(w io.Writer) (int64, error) {
 
 	return cw.Written(), err
 }
+
+func attachDbSecretValues(ctx context.Context, namespace string, k *kube.KubeConfig, dbValues *tt.TimDatabaseValues) {
```
Comment (anr-tonic) [inline cli/cmd/timstall/main.go:491]: This is kind of sloppy, but it works as a proof of concept for rehydrating the in memory values w/ what is stored in the cluster already.

**Comment by anr-tonic** [comment]
>I know it still draft, but tried running through this and it doesn't look like the secret name is being written to the values yaml.

I see it appearing at `web.configuration.database.secretName` w/ `timothy-db-access`

**Comment by anr-tonic** [comment]
Actually looking through the values file, should we also do this for the pull secret?

**Comment by daniel-tonic** [comment]
> Actually looking through the values file, should we also do this for the pull secret?

Maybe put that in another ticket to help keep the scope of this one down?  Not sure if the helm chart is already setup to support that or not.

**Comment by anr-tonic** [comment]
Closed in favor of #68

**Pull request closed without merging**